core/storage/base_model/gae_models.py

```diff
@@ -922,7 +934,7 @@ class BaseSnapshotMetadataModel(BaseModel):
     """
 
     # The id of the user who committed this revision.
-    committer_id = ndb.StringProperty(required=True)
+    committer_id = ndb.StringProperty(indexed=True, required=True)
```

Comment:
This seems like we need a separate PR for it plus a one-off job to re-put all models?

Reply:
Yeah, this will be done in separate PR.